ci-operator/step-registry/ipi/conf/inframachineset/ipi-conf-inframachineset-commands.sh

```diff
@@ -200,4 +197,4 @@ fi
 
 echo "Waiting for all pods to settle"
 sleep 5
-while [[ $(oc get pods --no-headers -A | grep -Pv "(Completed|Running)" | wc -l) != "0" ]]; do echo -n "." && sleep 5; done
+while [[ $(echo -e "openshift-ingress\nopenshift-image-registry\nopenshift-monitoring" | xargs -I {} oc get pods -n {} --no-headers | grep -Pv "(Completed|Running)" | wc -l) != "0" ]]; do echo -n "." && sleep 5; done
```

Comment:
Why are we now limiting this to certain namespaces, rather than waiting for all pods?

Reply:
when I triggled the job, it always failed at this check of the steps, about 2 times in 3, and when I debug the cluster status, even before this step start running, there are sometime several pods show Error but the new created pods are running well for the related operator, , and this step only deal with ingress, image-registry and monitoring operator, the error pods even didn't related the  changed operator. to make the CI more  robustness, just checking namespaces which doing changes rather than all namespace will make more sense in my option, what do you think?
and also all this failure is before my new added codes, machine type migration step is after it